dependencies.props

```diff
@@ -1,15 +1,18 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
   <PropertyGroup>
-    <CoreFxCurrentRef>368eeab3bf02cc55086e170d500cfbb15c974acc</CoreFxCurrentRef>
-    <CoreClrCurrentRef>f4e6d7f7e3c10cbd87d5d9e0fcee32e9aa19f6d1</CoreClrCurrentRef>
+    <CoreFxCurrentRef>534770cbd98c4083de4d93b809f2e0968f0f856a</CoreFxCurrentRef>
     <WCFCurrentRef>c9a03fe0c3309508cb509a4ffdd6dc8b79ed525e</WCFCurrentRef>
+    <StandardCurrentRef>8f8313cbe4b0f4d89c4d0835e08e3e340a18eec0</StandardCurrentRef>
   </PropertyGroup>
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25309-01</CoreFxExpectedPrerelease>
-    <StandardExpectedPrerelease>preview2-25309-01</StandardExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview2-25405-01</CoreFxExpectedPrerelease>
+    <StandardExpectedPrerelease>preview2-25401-01</StandardExpectedPrerelease>
+    <NETStandardPackageVersion>2.0.0-preview2-25401-01</NETStandardPackageVersion>
+    <NETStandardPackageId>NETStandard.Library</NETStandardPackageId>
+    <MicrosoftNETCoreAppPackageVersion>2.0.0-preview2-25407-01</MicrosoftNETCoreAppPackageVersion>
```

Comment:
I'd prefer `MicrosoftNETCoreAppPackageVersion` to be in the next PropertyGroup until it's actually auto-upgraded, just for clarity. (I'm ok with `NETStandardPackageId` because it's obviously static and just associated with `NETStandardPackageVersion`.)